src/test/java/de/rabea/game/BoardTest.java

```diff
@@ -0,0 +1,181 @@
+package de.rabea.game;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import static de.rabea.game.Mark.*;
+import static junit.framework.TestCase.assertEquals;
+import static junit.framework.TestCase.assertTrue;
+import static org.junit.Assert.assertArrayEquals;
+import static org.junit.Assert.assertFalse;
+
+
+public class BoardTest {
+
+    private Board board;
+
+    @Before
+    public void setup() {
+        board = new Board();
+    }
+
+    @Test
+    public void initiatesAnEmptyBoard() {
+        Mark[] emptyBoard = {EMPTY, EMPTY, EMPTY,
+                             EMPTY, EMPTY, EMPTY,
+                             EMPTY, EMPTY, EMPTY};
+        assertArrayEquals(emptyBoard, board.cells());
+    }
+
+    @Test
+    public void placesAMark() {
+        Mark[] cells = {EMPTY, X, EMPTY,
+                        EMPTY, EMPTY, EMPTY,
+                        EMPTY, EMPTY, EMPTY};
+        board.placeMark(1, X);
+        assertArrayEquals(cells, board.cells());
+    }
+
+    @Test
+    public void boardIsEmpty() {
+        assertFalse(board.isFull());
+    }
+
+    @Test
+    public void boardIsFull() {
+        board = fullBoardNoWinner();
+        assertTrue(board.isFull());
+    }
+
+    @Test
+    public void isChosenPositionStillFree() {
+        assertEquals(true, board.isPositionAvailable(1));
+    }
+
+    @Test
+    public void isNotAValidUserChoice() {
+        board.placeMark(1, X);
+        assertEquals(false, board.isPositionAvailable(1));
+    }
+
+    @Test
+    public void switchesMarkXtoO() {
```

Comment:
This test does not even need the `Board`. 
The functionality might be more at home somewhere else...